pixels-daemon/src/main/java/io/pixelsdb/pixels/daemon/scaling/policy/helper/FixedSizeQueue.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 PixelsDB.
+ *
+ * This file is part of Pixels.
+ *
+ * Pixels is free software: you can redistribute it and/or modify
+ * it under the terms of the Affero GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * Pixels is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * Affero GNU General Public License for more details.
+ *
+ * You should have received a copy of the Affero GNU General Public
+ * License along with Pixels.  If not, see
+ * <https://www.gnu.org/licenses/>.
+ */
+package io.pixelsdb.pixels.daemon.scaling.policy.helper;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class FixedSizeQueue
```

Comment:
Add a javadoc to each new class, explain the usage of the class (e.g., whether this is threadsafe, what should be taken care of...), and add the `@author` and `@create yyy-mm-dd`.